Shaders/DisplayDepth.fx

```diff
@@ -22,11 +22,21 @@
 uniform int iUIInfo <
 	ui_type = "combo";
 	ui_label = "How To Setting";
-	ui_tooltip = "Pick the value from 'Depth Input Settings'\n"
-	             "that lets the scene look the most natural.\n"
-	             "Then put the values from the tooltip into\n"
-	             "the settings.\n"
-	             "(Settings Tab -> Preprocessor Definitions)";
+	ui_tooltip = "This shader helps finding the right\n"
+                 "preprocessor settings for the depth\n"
+                 "input. By default the calculated normals\n"
+                 "are shown and the goal is to make the\n"
+                 "surfaces look smooth.\n"
+                 "Change the options for 'Reversed' and\n"
+                 "'Logarithmic' until this happens.\n"
+                 "\n"
+                 "When the right settings are found click\n"
+                 "'Edit global preprocessor definitions'\n"
+                 "(Variable editor in the 'Home' tab)\n"
+                 "and put them in there.\n"
+                 "\n"
+                 "In order to see the actual depth values\n"
+                 "change 'Show normals' to 'Show depth'";
```

Comment:
Forgot to mention this in the changelog, but 4.0 now supports "ui_tooltip" on techniques, so that when you hover one in the technique list, that tooltip text is displayed. I believe it makes sense to move this there, instead of using a dummy variable.
```technique DisplayDepth < ui_tooltip = "..."; >```